packages/@aws-cdk/aws-certificatemanager/README.md

```diff
@@ -1,4 +1,5 @@
 ## AWS Certificate Manager Construct Library
+
```

Comment:
this newline might break validation that a stability banner exists. 
might be something to reconcile in a different PR but I'd leave it alone for this one :)